renovate.json

```diff
@@ -2,7 +2,7 @@
   "extends": [
     "config:base"
   ],
-  "enabled": "true",
+  "enabled": true,
```

Comment:
Not sure it matters, since it worked with that before, but doesn't hurt to correct it. We can see from other boolean values that they're not quoted.

Reply:
Yes